crates/beerus-rpc/tests/common/mod.rs

```diff
@@ -129,6 +145,31 @@ fn mock_get_events() -> Mock {
             "application/json",
         ))
 }
+
+fn mock_add_invoke_transaction() -> Mock {
+    let invoke_transaction = BroadcastedInvokeTransaction::V0(BroadcastedInvokeTransactionV0 {
+        max_fee: FieldElement::ZERO,
+        signature: vec![],
+        nonce: FieldElement::ZERO,
+        contract_address: FieldElement::ZERO,
+        entry_point_selector: FieldElement::ZERO,
+        calldata: vec![],
+    });
+
+    let body = body_json(StarknetRpcBaseData::add_invoke_transaction([
+        &invoke_transaction,
+    ]));
+
+    println!("{body:#?}");
+
+    Mock::given(method("POST")).and(body).respond_with(
+        response_template_with_status(StatusCode::OK).set_body_raw(
+            include_str!("data/starknet_getEvents.json"),
```

Comment:
Hey @bigherc18 , I took the liberty of reviewing the problem you mentioned and found out that the wrong file was being used in this line. Instead of `data/starknet_getEvents.json`, it should be `data/starknet_addInvokeTransaction.json`. That's what was causing the error.

Also, I noticed some warnings about unused imports and some prints in the code.
By the way, it would be helpful to add an example in the `examples/beerus-rpc` folder for future reference.

Reply:
Great work @dpinones 🤝